test/unit/annotation_spec.js

```diff
@@ -233,6 +233,74 @@ describe('annotation', function() {
     });
   });
 
+  describe('TextAnnotation', function() {
```

Comment:
Could you place this `describe` block below the `AnnotationBorderStyle` `describe` block for a more logical ordering?

Reply:
Yes, I think that's even better. Let's go for that.